src/worker.py

```
@@ -1329,6 +1314,354 @@ async def serve_static(path: str, env):
+        except Exception as exc:
+            print(f"[ClassroomDO._session_for_ws.deserialize] error={exc!r}")
+
+        for sid, info in self.sessions.items():
+            try:
+                if info["ws"] == ws:
+                    return (sid, info)
+            except Exception as exc:
+                print(f"[ClassroomDO._session_for_ws.fallback] sid={sid} error={exc!r}")
+        return None
+
+    def _broadcast(self, msg, exclude_session_id=None):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_broadcast`

Add return type annotation: `None`

(ANN202)